resources/begrep/sikkerDigitalPost/nyinf/api/openapi_spec.yaml

```diff
@@ -1,173 +1,164 @@
 openapi: 3.0.2
 info:
-    title: API for aksesspunktleverandører i DPI
-    version: 0.0.1
-    description: API for kommunikasjon mellom C1 og C2 i firehjørers akritektur for DPI
+    title: API for digital post til innbygger (DPI)
+    version: 1.0.0
+    description: API for digital post til innbygger (DPI). APIet brukes av avsender (Peppol hjørne 1) og tilbys av avsenders aksesspunktleverandør (Peppol hjørne 2).
     x-logo: 
       url: "https://docs.digdir.no/assets/Digdir.png"
       altText: "Digdir logo"
 paths:
-    /{version}/send:    
+    /messages/out:
       post:
-        summary: Intitierer og laseter opp ny forsendelse til C2.
-        parameters: 
-            - in: path
-              name: version            
-              schema:
-                type: string                               
-                description: Versjon av API
-              required: true
+        summary: Send forsendelse til innbygger
+        parameters:
+          - in: query
+            name: kanal
+            schema:
+              type: string
+              description: Hvert avsendersystem for en gitt avsender bruker vanligvis en unik kanal (MPC) slik at avsendersystemene ikke leser hverandres innkommende meldinger. Innkommende meldinger i konversasjonen knyttet til den sendte meldingen blir markert med denne kanalen.
+            required: true
```

Comment:
Denne trenger ikke være påkrevet i API'et for vår del, optional er godt nok for oss (i praksis ville vi på API siden markert dem med `kanal = ukjent` derom det ikke var angitt noe)

Reply:
Men kanalnavn må ikke innhold whitespaces.   Det er lov med bokstaver, tall, underscore og minus.
Noen lovlige eksempler :
AGRESSO_LONN, SAP_REGNSKAP, INNKREVING, UBW_2024

Reply:
Ok, fint.

Kva med ØÆÅ ?

Case-insensitivt eller -sensitivt?